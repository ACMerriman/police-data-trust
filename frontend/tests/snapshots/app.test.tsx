import * as React from "react"
import Home from "../../pages/index"
import renderer from "react-test-renderer"

it("renders Home correctly", () => {
  const tree = renderer.create(<Home />).toJSON()
  expect(tree).toMatchSnapshot()
})
