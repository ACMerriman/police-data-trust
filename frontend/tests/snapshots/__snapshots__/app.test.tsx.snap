// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Home correctly 1`] = `
<h1>
  NPDC Home page
</h1>
`;
